src/rtcModule/rtcStats.cpp

```diff
@@ -319,8 +336,8 @@ void RtcStats::toJson(std::string& json) const
 {
     json.reserve(10240);
     json ="{";
-    JSON_ADD_STR(cid, mSessionId.toString());
-    JSON_ADD_STR(sid, mSessionId.toString());
+    JSON_ADD_STR(cid, mCallId.toString() + ":" + mSessionId.toString());
+    JSON_ADD_STR(sid, mCallId.toString() + ":" + mSessionId.toString());
```

Comment:
Review if sending `sid` is really required or not. Now that `sid` and `cid` values are both added to the `cid`, `sid` should be redundant.